### PR TITLE
Copy prior art from README

### PR DESCRIPTION
**Pull request by lassik**

### CODE REVIEW

File: srfi-alist.html [hunk 1]
```diff
@@ -66,6 +70,442 @@ <h2 id="rationale">Rationale</h2>
...
+          <td>
+            <p>pred</p>
+          </td>
+          <td>
+            <p>SLIB</p>
+          </td>
+        </tr>
+        <tr>
+          <td>
+            <p><strong>predicate→asso</strong></p>
```
Comment (mnieper) [inline srfi-alist.html:497]: The arrow looks strange.

Comment (lassik) [reply srfi-alist.html:497]: Ah, that's from the AsciiDoc converter which turns `->` arrows into Unicode arrows by default. I'll fix it.

**Comment by lassik** [comment]
I'll make another commit to add the References from the README as well.

**Comment by lassik** [comment]
OK, done.

**Pull request merged**